### DISCUSSION TITLE
2023-08-09 Weekly Meeting 119 Notes/Agenda

### DISCUSSION THREAD

**Discussion by mrmetaverse** [omigroup/omigroup#410] [General]
Weekly Meeting 119 is happening on Wednesday, August 9, 2023 from 9:00 AM PST (16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat.

Who we are and why we're here:
The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal.

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

Agenda

I'd like us to build things together. I also have to get with Jin to talk metaverse.dev, and need to relay the discussions from last Fridays champions sync: new chair election, Open Collective Update

- Metaverse Festival October 2024 see [HackMD draft information for page on OMI website to start spreading the word](https://hackmd.io/@indiebio/metaverse-float-parade)

**UPDATES**

comments below

**Comment by indiebio** [comment]
OMI is an W3C accredited research body

**Reply by indiebio** [reply]
need to update W3C page, need to "merge" or coordinate the changes between how we represent ourselves there. align the gateways, how people find us

**Reply by indiebio** [reply]
https://github.com/omigroup/omigroup/blob/main/CHARTER.md

**Comment by indiebio** [comment]
Elections soon. There does not seem to be formal documentation about the chairs/"leadership"? of OMI. We should have something. Some resources shared by the group to help with this:
- https://hackmd.io/@mrmetaverse/omi-tasks-audit (Edit: this basically covers it. but would be good to look at e.g. BigScience and see if it's worth updating)
- https://discord.com/channels/770382203782692945/834825299438731325
- THese guys have a well articulated govenership for open source collaboration orgs  .....   https://bigscience.notion.site/BigScience-214dc9a8c1434d7bbcddb391c383922a

**Comment by mrmetaverse** [comment]
Call for Chair Nominations: 

- we have one vacant chair seat. 
- champions feel we should hold a new election, as availabilities have changed, and we have some new contributors who have more time to contribute. 
- nominate yourself or others in the comments below :point_down: 

re: https://github.com/omigroup/omigroup/discussions/410#discussioncomment-6682492

**Reply by indiebio** [reply]
here goes :) 
I want to nominate myself. My idea of the metaverse in a nutshell is "layers and layers of interactive information" (3D data visualisation and then some, the 'some' including metadata). I am also interested in the potential of the internet to improve democracy, governance and accountability. This links to conversations about digital identity, trust, and community. 

Through these interests and my background I consider myself to be well suited to the core values of OMI, particularly to "collaborate, driven by research, privacy and accessibility, to make the metaverse more human."

In terms of diversity, I am a good fit. While currently in Portugal I am originally from South Africa and am passionate about the internet and its next iteration working for people from all backgrounds, particularly those held back by economic, political or geographical constraints. Further, I am not a developer, so while I am blown away by the stuff jin shows every week and the potential of virtual worlds, it is very strange and overwhelming to me. This puts me in a good position to imagine what new people would see, even as it makes it hard for me to keep up sometimes :)

The project I am working on (why I'm here, WIIFM) is [peduncle](https://peduncle.world/) with emphasis on the interoperability and decentralisation of structured data (technically speaking my interest is in (grass-roots) urban resource management), and so I do consider the metaverse as the visual extension of Sir Tim Berners-Lee's idea of the next version of the internet.

Having read [#157](https://github.com/omigroup/omigroup/discussions/157) , [#165](https://github.com/omigroup/omigroup/discussions/165) and the [task audit](https://hackmd.io/@mrmetaverse/omi-tasks-audit) I fully agree! My view is that chairs provide continuity, maintaining the values and transferring institutional knowledge. The working groups, tasks and daily responsibilities runs parallel/independent to that, but practically chairs do probably catch a lot of that. Good leadership could be interpreted as empowering others to feel confident in contributing to those.

I am currently unclear on the differences between "Steering Committee roles" and "[W3C](https://github.com/omigroup/omigroup/blob/main/CHARTER.md) chairs."

**Comment by indiebio** [comment]
jin proposed getting a Fireflies subscription to help with archiving and note taking: https://hackmd.io/@XR/omi-fireflies. 
Formal proposal: https://github.com/omigroup/omigroup/issues/411

**Comment by mrmetaverse** [comment]
# Vote passed: Fireflies 
![image](https://github.com/omigroup/omigroup/assets/63426722/d95dcabc-5e9f-4039-9a85-4abbc1e2d385)

**Comment by indiebio** [comment]
https://sketchfab.com/3d-models/hyperfy-simple-cars-parade-float-d1a2b04bc8684e31a2c59923630ec516

test world for the festival (I think) https://hyperfy.io/cars

**Comment by indiebio** [comment]
Follow up from last week: media meeting: Tuesday (AUG15)  16:00 - 17:00 UTC

Recurring meeting is on the second Tuesday every month at 16:00 - 17:00 UTC (24:00 Format) to accommodate MSF commitments. .....?

https://github.com/omigroup/media

and follow up about recording links - fireflies should fix where to find these, streamline this.

**Comment by mrmetaverse** [comment]
UPDATE: We were approved to join OCF under 501c3 fiscal sponsorship. But still need to work out the zeroing the balance thing. 

![image](https://github.com/omigroup/omigroup/assets/63426722/42589a9e-96c8-49ea-8bfe-4e9023f0d876)
